test/integration/target/withdraw-multiple-to-recipient/withdrawMultipleToRecipient.t.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.19 <0.9.0;
+
+import { ISablierV2Lockup } from "@sablier/v2-core/src/interfaces/ISablierV2Lockup.sol";
+
+import { Errors } from "src/libraries/Errors.sol";
+
+import { Target_Integration_Test } from "../Target.t.sol";
+
+abstract contract WithdrawMultipleToRecipient_Integration_Test is Target_Integration_Test {
```

Comment:
Let's use the plural form in this name, i.e., `WithdrawMultipleToRecipients`.

Reply:
Don't understand why, can you explain?

Reply:
The sender's streams could be associated with multiple, different recipients.